### PR TITLE
Update imported package name-- Activity feature computation

### PR DESCRIPTION
**Pull request by sakther**
This change is [Reviewable](https://reviewable.io/reviews/md2korg/cerebralcortex-dataanalysis/19)

### CODE REVIEW

File: core/feature/activity/activity.py [hunk 1]
```diff
@@ -23,101 +24,111 @@
...
     def process(self):
         if self.CC is not None:
-            print("Processing ActivityMarkers")
-            self.all_users_data("mperf")
+            all_users = self.CC.get_all_users("mperf")
 
+            for user in all_users:
+                user_id = user["identifier"]
+
+                streams = self.CC.get_user_streams(user_id)
```
[the remark this answers is not in the document]
Comment (anandatirtha) [reply core/feature/activity/activity.py]: Handle for empty return values

File: core/feature/activity/ACTIVITY_CONSTANTS.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+study_name = 'mperf'
```
[the remark this answers is not in the document]
Comment (sakther) [reply core/feature/activity/ACTIVITY_CONSTANTS.py]: I used it to get users from CC

[the remark this answers is not in the document]
Comment (sakther) [reply core/feature/activity/ACTIVITY_CONSTANTS.py]: Moved

File: core/feature/activity/activity.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (sakther) [reply core/feature/activity/activity.py]: Handled empty return values

**Comment by sakther** [comment]
Hi Anand,
I updated the pull request.

Regards,
Sayma

**Pull request closed without merging**